build.gradle.kts

```diff
@@ -14,39 +14,34 @@
  * limitations under the License.
  */
 import org.asciidoctor.gradle.jvm.AsciidoctorTask
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
 
     // project plugins
-    `java-gradle-plugin`
```

Comment:
Why the `java-gradle-plugin` and `maven-publish` plugins are dropped?

Reply:
Since 1.0.0 they are automatically applied by the gradle publish plugin: https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.0.0